api/app/Console/Commands/UpdateApplicationTimestamps.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Enums\PoolCandidateStatus;
+use App\Models\PoolCandidate;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Builder;
+
+class UpdateApplicationTimestamps extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:update-application-timestamps
+                            {--dry : Dry run to see how many candidates will be affected}
+    ';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
```

Comment:
If we are going to have a description it should at least say something useful.

Reply:
Done! [update command description](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12734/commits/c80b60caac82d335a088f5f1ec6629fb9114a25f)